app/models/data_export.rb

```diff
@@ -7,19 +7,17 @@ class DataExport < ApplicationRecord
 
   belongs_to :organization
   belongs_to :membership
+  has_one :user, through: :membership
 
   has_one_attached :file
 
   has_many :data_export_parts
 
-  validates :resource_type, presence: true
-  validates :format, presence: true, inclusion: {in: EXPORT_FORMATS}
-  validates :status, presence: true, inclusion: {in: STATUSES}
```

Comment:
These validations were not tested and in fact they don't even have change to work because of specific to enum logic.
```ruby
irb(main):003> data_export = DataExport.new => #<DataExport:0x0000ffff5a6eed80
irb(main):004> data_export.status = 'no-existing-one'
(irb):4:in `<main>': 'no-existing-one' is not a valid status (ArgumentError)

          raise ArgumentError, "'#{value}' is not a valid #{name}"
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

With suggested changes
```ruby
irb(main):001> data_export = DataExport.new => #<DataExport:0x0000ffff7f9b5648
irb(main):002> data_export.status = 'no-existing-one'
=> "no-existing-one"
irb(main):003> data_export.valid?
=> false
irb(main):004> data_export.errors.as_json
=> 
{:organization=>["relation_must_exist"],
 :membership=>["relation_must_exist"],
 :format=>["value_is_invalid"],
 :status=>["value_is_invalid"],
 :resource_type=>["value_is_mandatory"]}
```